Message-Id: <87e0b605aa093d95c507083de601261098cd1aff.1336068799.git.agx@sigxcpu.org>
In-Reply-To: <e82f2035cd9341f3530199e68c11a27790f041de.1336068799.git.agx@sigxcpu.org>
References: <5909887366833c92178e3ed650b87d666f9f50c1.1336068799.git.agx@sigxcpu.org>
	<56db9d711b7615ece3848c57f71f362f0e5f4fea.1336068799.git.agx@sigxcpu.org>
	<727a1c60fdc68e24b2f457f5b7e3611b2aae7a7b.1336068799.git.agx@sigxcpu.org>
	<af426bb880140c2941366a3a84fd11a714b9c778.1336068799.git.agx@sigxcpu.org>
	<64360f6da8f2d82618e0eda5d971ee6a083851ae.1336068799.git.agx@sigxcpu.org>
	<308046ae1d09cc7cce80fdb268cfb7fee35de13d.1336068799.git.agx@sigxcpu.org>
	<f96b7f473f61a8e6d4866a49d9fcc861bed443b5.1336068799.git.agx@sigxcpu.org>
	<7e773c9b5d95a42321be6b5fc7a11cf0fd51a124.1336068799.git.agx@sigxcpu.org>
	<6d5763889d5f114ed6f149ecd47c562f06868519.1336068799.git.agx@sigxcpu.org>
	<e82f2035cd9341f3530199e68c11a27790f041de.1336068799.git.agx@sigxcpu.org>
From: =?UTF-8?q?Guido=20G=C3=BCnther?= <agx@sigxcpu.org>
Date: Sat, 2 Apr 2011 10:37:16 +0200
Subject: Brand it

---
 it/locale/it/lightning/lightning.dtd        |    2 +-
 it/locale/it/lightning/lightning.properties |    2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/it/locale/it/lightning/lightning.dtd b/it/locale/it/lightning/lightning.dtd
index 62c9501..0999394 100644
--- a/it/locale/it/lightning/lightning.dtd
+++ b/it/locale/it/lightning/lightning.dtd
@@ -40,7 +40,7 @@
    - the terms of any one of the MPL, the GPL or the LGPL.
    -
    - ***** END LICENSE BLOCK ***** -->
-<!ENTITY lightning.taskLabel "Lightning">
+<!ENTITY lightning.taskLabel "Iceowl">
 <!ENTITY lightning.menupopup.new.event.label "Evento…">
 <!ENTITY lightning.menupopup.new.event.accesskey "E">
 <!ENTITY lightning.menupopup.new.task.label "Attività…">
diff --git a/it/locale/it/lightning/lightning.properties b/it/locale/it/lightning/lightning.properties
index 33a1aea..195ac63 100644
--- a/it/locale/it/lightning/lightning.properties
+++ b/it/locale/it/lightning/lightning.properties
@@ -62,7 +62,7 @@ imipBarRequestText = Questo messaggio contiene un invito ad un evento.
 imipBarUpdateText = Questo messaggio contiene un aggiornamento per un evento esistente.
 imipBarAlreadyProcessedText = Questo messaggio contiene un evento che è stato già processato.
 imipBarReplyText = Questo messaggio contiene una risposta ad un invito.
-imipBarUnsupportedText = Questo messaggio contiene un invito che questa versione di Lightning non può processare.
+imipBarUnsupportedText = Questo messaggio contiene un invito che questa versione di Iceowl non può processare.
 imipBarProcessingFailed = Elaborazione del messaggio fallita. Stato: %1$S.
 imipAcceptInvitation.label = Accetta
 imipCancelInvitation.label = Elimina
